examples/nanoChatGPT/env.py

```diff
@@ -19,22 +19,24 @@
 @torch.no_grad()
 def _step(self, tensordict):
     self.step_num += 1
-    prompt = tensordict["prompt"]
+    prompt = tensordict.get("prompt")
 
     # perform the action
-    action = tensordict["action"].squeeze(-1)
+    action = tensordict.get("action").squeeze(-1)
 
     # compute the reward
-    if self.step_num >= self.config["episode_length"]:
-        reward = self.reward_model(prompt).unsqueeze(-1)
```

Comment:
Why did we unsqueeze? We end up with a reward of shape `[B, 1, 1]` when we set the spec to be `[B, 1]`